packages/core/src/agent/context/DefaultAgentContextProvider.ts

```diff
@@ -19,6 +19,10 @@ export class DefaultAgentContextProvider implements AgentContextProvider {
     this.agentContext = agentContext
   }
 
+  public async getDefaultAgentContext(): Promise<AgentContext> {
+    return this.agentContext
+  }
+
```

Comment:
I would like to avoid adding this, which should by possible If I saw the issue with the did resolver

```suggestion
```